recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -8,6 +8,7 @@ set PYTHONUNBUFFERED=1
 conda.exe config --set show_channel_urls true
 conda.exe config --set auto_update_conda false
 conda.exe config --set add_pip_as_python_dependency false
+conda.exe config --set channel_priority strict
```

Comment:
If we got rid of this for the moment, it would probably be OK

Reply:
No, osx has issues with libcxx as well.

Reply:
That is "fixed" with the choice of vendorizing the old `libcxx` and may be solved for good with https://github.com/conda/conda/pull/8961.

Reply:
That's a terrible fix as I mentioned in another issue that I will revert soon. Correct fix is https://github.com/AnacondaRecipes/aggregate/pull/162

Reply:
> If we got rid of this for the moment, it would probably be OK

Can you elaborate why @scopatz? If we can list the foreseeable issues on Windows I can start working on them. (BTW, I've been using strict on Windows and Linux for a while now with more success than macOS, which should also be OK-ish now.)